exercise-1.js

```diff
@@ -252,18 +375,31 @@
         // Square every number in the array.  Store the squares in a new array.
 
         //your code...
-
-
+        function squareAll(nums){
+            squaredArray = [];
+            for(let i = 0; i < nums.length; i++){
+                squared = Math.pow(nums[i],2);
+                squaredArray.push(squared);
+            }
+            return squaredArray;
+        }
+        console.log(squareAll(nums));
 
 
 
         const fivePlus = [1,3,5,7,9,1,3,5,2,3,1,23,4,232,3,4,1,2,2,2,3,4,4,1,12,11,23,3,4,5];
         //Remove all numbers that are less than 5.  Store the results in a new array.
 
         //your code...
-
-
-
+        function removeAll(array){
+            newArray = [];
+            for(let i = 0; i < array.length; i++){
+                if (array[i] >= 5) {
+                    newArray.push(array[i]);
+                }
+            } return newArray;
+        }
```

Comment:
Again, this solution isn't using any Array methods, which would greatly simplify the code.